### ISSUE TITLE
tcltk-wish.sh: command not found

### ISSUE THREAD

**Issue by rnkn** [pure-data/pure-data#96]
Hi @danomatika I think there's a tiny error in `osx-app.sh` on [line 173](https://github.com/pure-data/pure-data/blob/f8f8d4252d28110b0fd4674cc21819190905b247/mac/osx-app.sh#L173).

I think it should be:
~~~diff
@@ -170,7 +170,7 @@ if [ $included_wish == true ] ; then
 elif [ "$WISH" == "" ] ; then
     if [ "$TK" != "" ] ; then
         echo "Using custom $TK Wish.app"
-        tcltk-wish.sh $universal $TK
+        ./tcltk-wish.sh $universal $TK
         WISH=Wish-${TK}.app
     elif [ "$SYS_TK" != "" ] ; then
         echo "Using system $SYS_TK Wish.app"
~~~

**Comment by umlaeute** [comment]
more likely `${0%/*}/tcltk-wish.sh`

**Comment by danomatika** [comment]
Not necessarily. At this point, the working directory is the script directory aka `mac` where both scripts reside.

**Comment by umlaeute** [comment]
i'll not push a change then :-)

**Comment by danomatika** [comment]
I'm noticing another issue related to the build dir detection as well. Fixing soon.

**Comment by danomatika** [comment]
This is fixed.

**Comment by rnkn** [comment]
Just pointing out that now the script cannot be run from `./osx-app.sh` (thinks Pd hasn't been built) and must be run from `mac/osx-app.sh` from the parent directory.

**Comment by danomatika** [comment]
*sigh* This is what I get for trying to make it "smart."

**Comment by danomatika** [comment]
Ok, should be fixed now in #88
